builder/googlecompute/config.go

```diff
@@ -121,7 +121,7 @@ type Config struct {
 	// If true, launch a preemptible instance.
 	Preemptible bool `mapstructure:"preemptible" required:"false"`
 	// The time to wait for instance state changes. Defaults to "5m".
-	RawStateTimeout string `mapstructure:"state_timeout" required:"false"`
+	StateTimeout time.Duration `mapstructure:"state_timeout" required:"false"`
```

Comment:
These `RawXxx` cleanups/changes came from my first iteration; they don't change how the code behaves but just remove some code. I figured I'd let them stay here 🙂

Reply:
Meaning if you want me to revert; I can !